crates/pool/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
why can't we add this to `chain` crate? We're trying to limit the amount of crates

Reply:
I though `chain` crate was related only to block handling and since the mempool will most likely get pretty complex I created a new crate for it.
Anyways, I moved it to the `chain` crate.